src/graphql-schema.test.js

```diff
@@ -15,8 +15,9 @@ beforeEach(async () => {
   getHost = () => `http://127.0.0.1:${port}`
 })
 
-afterEach(async () => {
+afterAll(async (done) => {
   await app.close()
+  done()
```

Comment:
Is that some jest/mocha magic?

Reply:
Yes, but I don't think it will work. because of this [graphql-yoga](https://github.com/prisma/graphql-yoga/issues/404) you can't close the server from `app` you can only close it in the `server variable` according to the link

Reply:
The `done` callback is normally used to test async behaviour and callbacks. You can call `done()` in your callback and the test will not pass until it `done` reached. If a test case has no `done` in the method argument list, the method body will be executed and the test will pass if no error was raised. To prevent this you can use `done` callback.

If this does not fix the build, can we maybe revert it?

@Kachulio1 `git revert 3a9cb31e1f6039aecd428b2fc250b4b88a18ccfa` ?

Reply:
I did that just now